src/twisted/topfiles/8941.feature

```diff
@@ -0,0 +1 @@
+twisted.internet.reactor.spawnProcess now does not raise a deprecation warning on Unicode arguments. It will encode Unicode arguments down to bytes using the filesystem encoding on UNIX and Python 2 on Windows, and pass Unicode through unchanged on Python 3 on Windows.
```

Comment:
The right language here would be "emit", not "raise".  "raise" should generally be reserved for exceptions.